scaleway/resource_tem_domain.go

```diff
@@ -140,6 +170,12 @@ func resourceScalewayTemDomainRead(ctx context.Context, d *schema.ResourceData,
 	_ = d.Set("last_error", domain.LastError)
 	_ = d.Set("spf_config", domain.SpfConfig)
 	_ = d.Set("dkim_config", domain.DkimConfig)
+	_ = d.Set("smtp_host", "smtp.tem.scw.cloud")
+	_ = d.Set("smtp_port_unsecure", 25)
```

Comment:
Anyhow, if the product API cannot return those constants, they should live in the SDK as they are constant regardless of the devtools using them.